Tools/Docusaurus/docs/Setup.md

```diff
@@ -3,15 +3,34 @@
 **Requirements**
 - [Unity 2019.4.31f1](https://unity3d.com/get-unity/download/archive)
 - [VRCSDK3 + Udon](https://vrchat.com/home/download)
-- [UdonSharp](https://github.com/Merlin-san/UdonSharp/releases)
 
 **Installation**
 
-> NOTE: This will be easier soon with the release of the VRChat Creator Companion. These notes are for the legacy ".unitypackage" format.
+You can get UdonSharp by using the [VRChat Creator Companion](https://vcc.docs.vrchat.com/) also known as the VCC, its [CLI](https://vcc.docs.vrchat.com/vpm/cli/), or a [starter template](https://github.com/vrchat-community/template-udonsharp). 
 
-- Read the official [Getting Started With Udon](https://ask.vrchat.com/t/getting-started-with-udon/80) thread, this has basic installation instructions for Udon.
-- Install the latest version of [VRCSDK3 + Udon](https://vrchat.com/home/download).
-- Get the [latest release of UdonSharp](https://github.com/Merlin-san/UdonSharp/releases) and install it in your project.
+## Create a new UdonSharp project with the VCC:
+- Install the latest version of the [Creator Companion](https://vrchat.com/home/download).
+- From the main screen, select "New", then "UdonSharp", and choose a directory.
+- Press "Open Project". That's it!
+
+## Create a new UdonSharp Project with Source Control:
+- Visit the [UdonSharp Project Template repo](https://github.com/vrchat-community/template-udonsharp).
```

Comment:
Do you think we should avoid abbreviations such as 'repo'? Knowledge about Git is only required for certain sections of the guide, and readers may be unfamiliar with the term.
```suggestion
- Visit the [UdonSharp Project Template repository](https://github.com/vrchat-community/template-udonsharp).
```